KinveyResearchKit/TaskResult.swift

```diff
@@ -29,7 +29,7 @@ open class TaskResult: CollectionResult {
     override open func propertyMapping(_ map: Map) {
         super.propertyMapping(map)
         
-        taskRunUUID <- map[PersistableIdKey]
+        taskRunUUID <- map["taskRunUUID"]
```

Comment:
don't we should keep the `_id`? do we need to enforce the `taskRunUUID` to be unique?